### PR TITLE
Competitive Landscape

### PR DESCRIPTION
**Pull request by bbengfort**
This PR implements a partial for the competitive landscape table and adds all the data from the spreadsheet that @rebeccabilbro created. It also removes the "request alpha access" form from the Ensign page and invites users to join the Beta study. 

The first draft of this table is:

Screenshot 2023-05-02 at 18 00 33

Now to make it look better and more like the spreadsheet!

### CODE REVIEW

**Comment by bbengfort** [comment]
Here's what it looks like now:

Screenshot 2023-05-02 at 18 44 45

Not perfect, but publishable … happy for any feedback or thoughts.

File: data/en/ensign.yaml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    subtitle: By default, replay all events from Day 1.
+  - id: consistency
+    title: Global Consistency
+    subtitle: All users can count on the same data, regardless of where they live.
+  - id: beginner
+    title: Beginner-Friendly
+    subtitle: Quickstarts, SDKs, and templates.
+  - id: neckbeard
+    title: Neckbeard-Friendly
+    subtitle: Fine controls, transparency, &amp; granularity.
```
Comment (daniellemaxwell) [inline data/en/ensign.yaml:19]:
```suggestion
    subtitle: Fine controls, transparency, & granularity.
```

**Review by daniellemaxwell — APPROVED** [review]
The table looks really good. I was going to suggest possibly changing the border color at the top to stand out more, but I tried a few color options and it made the table look bad. I have one question about whether a couple of links should be enclosed with button tags, since they appear as buttons on the screen. The rest looks fine, so it's approved.

**Review by rebeccabilbro — APPROVED** [review]
This looks so good @bbengfort! The only *tiny* thing that I think I'd change is that we not categorize Ensign under the "Eventing Tools" (since we're going to try to argue that it's something between a DB and an eventing tool)
Screen Shot 2023-05-03 at 1 28 52 PM

I'm not quite sure what the best way would be to draw a visual distinction, but maybe something like the above?

File: layouts/partials/ensign-beta.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <a href="https://github.com/rotationalio/go-ensign" target="_blank" class="text-[#1D65A6] font-bold">Go</a>,
+    <a href="https://github.com/rotationalio/watermill-ensign" target="_blank" class="text-[#1D65A6] font-bold">Watermill</a>,
+    or <a href="https://github.com/rotationalio/pyensign" target="_blank" class="text-[#1D65A6] font-bold">Python SDKs</a>
+    to get started building your first event-driven application. We know your
+    time is valuable, but these are the early stages for Ensign and you could
+    dramatically shape what it becomes. Take part in our
+    <a href="https://forms.gle/V2Gt48CJttFGitGZ8" target="_blank" class="text-[#1D65A6] font-bold">User Study</a>
+    and we'll send you a special thank you gift!
+  </p>
+  <p class="text-center text-white font-semibold pt-5 pb-10">
```
[the remark this answers is not in the document]
Comment (bbengfort) [reply layouts/partials/ensign-beta.html:17]: The `<p>` is holding 2 `<a>` tags which are the buttons; but I changed it to a `<div>` to make it more semantically correct.

File: layouts/partials/competitive-landscape.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  <h2 class="pb-5 leading-normal font-bold text-3xl">
+    Competitive Landscape
+  </h2>
+  <p class="pb-5">
+    How does Ensign stack up to other, similar tools and products?
+  <p>
+
+  <table class="table-auto border-collapse border border-slate-600 w-full text-md">
+    <thead>
+      <!-- TODO: not sure how to extract categories from the data yet -->
```
[the remark this answers is not in the document]
Comment (bbengfort) [reply layouts/partials/competitive-landscape.html:15]: Great idea! If we have multiple categories in the future; this is the way I'll go!

**Pull request merged**